### PR TITLE
Fixed issue with ArgumentError and time formatting

### LINKED ISSUE #7265: Time.at argument handling
**Environment Information**
  
Provide at least:
* JRuby version (`jruby -v`) and command line (flags, JRUBY_OPTS, etc)
* Operating system and platform (e.g. `uname -a`)

```sh
ruby -v
jruby 9.3.6.0 (2.6.8) 2022-06-27 7a2cbcd376 OpenJDK 64-Bit Server VM 18.0.1+10 on 18.0.1+10 +jit [arm64-darwin]
```

```sh
uname -a
Darwin laptop.local 21.5.0 Darwin Kernel Version 21.5.0: Tue Apr 26 21:08:37 PDT 2022; root:xnu-8020.121.3~4/RELEASE_ARM64_T6000 arm64
```

**Expected Behavior**

ruby 2.6.10p210 (2022-04-12 revision 67958) [arm64-darwin21]

```ruby
irb(main):001:0> Time.at(12345, 500, in: '-04:00')
=> 1969-12-31 23:25:45 -0400
```

**Actual Behavior**

```ruby
irb(main):001:0> Time.at(12345, 500, in: '-04:00')
Traceback (most recent call last):
       10: from /Users/---/.asdf/installs/ruby/jruby-9.3.6.0/bin/irb:23:in `<main>'
        9: from org/jruby/RubyKernel.java:1052:in `load'
        8: from /Users/---/.asdf/installs/ruby/jruby-9.3.6.0/lib/ruby/gems/shared/gems/irb-1.0.0/exe/irb:11:in `<main>'
        7: from org/jruby/RubyKernel.java:1237:in `catch'
        6: from org/jruby/RubyKernel.java:1237:in `catch'
        5: from org/jruby/RubyKernel.java:1507:in `loop'
        4: from org/jruby/RubyKernel.java:1091:in `eval'
        3: from (irb):1:in `evaluate'
        2: from org/jruby/RubyTime.java:1371:in `at'
        1: from org/jruby/RubyTime.java:1360:in `at'
ArgumentError (unexpected unit )
```

The default unit for this form of call is apparently microseconds.

### PR DESCRIPTION
**Pull request by PurityLake**
Fixes #7265 

`RubyTime.atMulti` would be passed `arg3` as `nil` if it wasn't given by the user. It complained that the unit couldn't be found.

So I added a check to see if it is `nil`, if it is then it is set to a symbol `:microsecond`. This appears to be the default value in CRuby.

Also updated the formatting as it wasn't the same as Ruby 3.

My Ruby Version: ruby 3.0.2p107 (2021-07-07 revision 0db68f0233) [x86_64-linux-gnu]
Example Code:
```ruby
Time.at(12345, 500, in: '-04:00')
```

Ruby:
```
1969-12-31 23:25:45.0005 -0400
```

JRuby after PR:
```
1969-12-31 23:25:45.0005 -0400
```

### CODE REVIEW

**Comment by PurityLake** [comment]
I removed the format specifiers for Time due to failing tests

File: core/src/main/java/org/jruby/RubyTime.java [hunk 1]
```diff
@@ -899,7 +899,7 @@ public RubyString asctime() {
     @JRubyMethod
     public IRubyObject to_s() {
         DateTimeFormatter simpleDateFormat = isUTC() ? TO_S_UTC_FORMATTER : TO_S_FORMATTER;
-
+
```
Comment (headius) [inline core/src/main/java/org/jruby/RubyTime.java:902]: Remove rogue whitespace; formatting standards for our codebase have no whitespace on blank lines (other than the newline).

**Review by headius — CHANGES_REQUESTED** [review]
Minor formatting change but otherwise good.

**Comment by headius** [comment]
Looks good!

**Pull request merged**